microsoft-edge/webview2/concepts/webview-features-flags.md

```diff
@@ -0,0 +1,152 @@
+---
+title: WebView2 browser flags
+description: Overview of browser flags (arguments, or switches) that affect the WebView2 control, for testing forthcoming features or diagnosing issues.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: webview
+ms.date: 02/14/2024
+---
+# WebView2 browser flags
+
+One of the ways you can interact with Microsoft Edge WebView2 and impact behavior is by passing browser flags (arguments, or switches) to WebView2.  These browser flags are useful for testing forthcoming features, and for diagnosing issues.  
+
+> [!WARNING]
+> Apps in production shouldn't use WebView2 browser flags, because these flags might be removed or altered at any time, and aren't necessarily supported long-term.
+
+Generally, the flags are owned by both Chromium and Microsoft Edge.  Chromium flags are not owned or controlled by Microsoft Edge, so Microsoft Edge doesn't have control over when or how the flags are altered in their behavior or removed.
+
+You can set browser flags in your local device environment, or set browser flags programmatically through code.  Both approaches are described below.
+
+
+<!-- ====================================================================== -->
+## Setting browser flags in your local device environment
+
+To test forthcoming features or to diagnose issues, we recommend using browser flags in your local device environment, via setting the `WEBVIEW2_BROWSER_ARGUMENTS` environment variable or via registry keys.  For more information, see the following Win32 API Reference: [CreateCoreWebView2EnvironmentWithOptions](/microsoft-edge/webview2/reference/win32/webview2-idl#createcorewebview2environmentwithoptions).
+
+
+<!-- ====================================================================== -->
+## Setting browser flags programmatically through code
+
+Or, you can set browser flags programmatically, by passing the browser flags as the `AdditionalBrowserArguments` property of `CoreWebView2EnvironmentOptions`:
```

Comment:
review this section - are the links incorrect except that the Win32 CreateCoreWebView2EnvironmentWithOptions link is correct?

https://learn.microsoft.com/microsoft-edge/webview2/reference/win32/webview2-idl#createcorewebview2environmentwithoptions

Reply:
A commit, which I can't find, added text like the following, that mentions "the AdditionalBrowserArguments API" - so I'm trying to construct a 3-tab tab-set containing API links for .NET/WinRT/Win32.

* Please review whether I put the correct API items in this tab-set, so that users can get to the API docs about the AdditionalBrowserArguments API.  
* Do we need to recommend that all wv2 devs for all platforms see the Win32-specific API Ref for CreateCoreWebView2EnvironmentWithOptions?
https://learn.microsoft.com/microsoft-edge/webview2/reference/win32/webview2-idl#createcorewebview2environmentwithoptions
Does that Win32 API Ref contain more relevant info than .NET or WinRT Ref docs?

Reply:
Let's do the 3-tab-set to cover all the platforms

Reply:
keeping as-is